### ISSUE TITLE
License - SDK - and schtuff

### ISSUE THREAD

**Issue by SveSop** [SveSop/wine-nvoptix#10]
I have not heard back from nVidia regarding the use of parts of the OptiX headers, but happened to fall in on this one: 
[https://forums.developer.nvidia.com/t/optix-scriptable-installation/71068/4](https://forums.developer.nvidia.com/t/optix-scriptable-installation/71068/4)

> The main licensing constraint is we ask for a formal notification if you use or distribute parts of OptiX with a commercial application. It is free to use and okay to share whatever you need in any case, under the terms of the license

Since WINE-NVOPTIX in no way is a commercial application, and we are not directly distributing the whole SDK... AND i have not heard back from nVidia (almost 2 weeks now.. have no idea the reply-time on developer inquiries).

@Saancreed @PetitMote
What do you think about this... just push "Release 1.0" on main, and cross our fingers, or want to hold off a bit longer?
And what about Release.. prefer anything in particular? v1.0 - v0.1 - R1.0 ??

**Comment by Saancreed** [comment]
That looks promising. I'd say let's push what we have to `master` and make a release (I have no preferences w.r.t. version here), there's not much difference between stuff being kept on `master` and some other branch.

Also fwiw [Arch is also waiting for NV](https://bugs.archlinux.org/task/72921) to clarify redistribution requirements of Optix headers for the purposes of building Blender with Optix integration enabled, but our case might be a little different, considering that we need a small part of multiple versions of Optix SDK instead of full copy of a single one.

**Comment by jp7677** [comment]
Regarding tag and release names, I would aim to be consistent with wine-nvml, thus first version would `v0.1`. I would not use 1.0 since you never know when you are done, a new optix version will likely require a new wine-nvoptix release.  Just increasing the minor version over time keeps expectations limited :)

PS: I would also push all to master/main.

**Comment by SveSop** [comment]
Sounds reasonable with the `v0.1` release to keep it somewhat in line with wine-nvml or whatnot.

@Saancreed Am i to understand that Blender needs optix headers to compile to have OptiX support then?

**Comment by Saancreed** [comment]
Yes, see the PKGBUILD for details. Previous versions included `https://developer.download.nvidia.com/redist/optix/v7.0/OptiX-7.0.0-include.zip` as one of source packages required at build time but it seems that this package is now too old for Blender and NV hasn't pushed 7.4.0 package anywhere yet (where it can be freely downloaded from, that is).

**Comment by SveSop** [comment]
Oki.. published release v0.1 : [https://github.com/SveSop/wine-nvoptix/releases/tag/v0.1](https://github.com/SveSop/wine-nvoptix/releases/tag/v0.1)

Gonna keep the "working-version" branch for a bit... we can always make PR's against that for testing purposes perhaps? Will see.. 

Keeping this issue open if or until any news from nVidia regarding the license usage. (or discussions thereof)

**Comment by jp7677** [comment]
@SveSop the release looks cool! Note that the correct way of writing `NVIDIA` is with all letters capitalized, not just the V. It looks better if you correct this in the readme and release notes.
